File: PDF1_Ex2.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int a=0, b=1, c=0, verif=1;
    int respa=0, respb=0, respc=0;
    for(a=1; a<1000; a++)
        for(b=a+1; b<1000; b++){
            c=1000-a-b;
            if(a*a+b*b==c*c){
                respa = a;
                respb = b;
                respc = c;
            }
        }
    printf("%d", respa*respb*respc);
    return 0;
}

File: PDF1_Ex2.py
respa=0		#Armazena a resposta de a.
respb=0		#Armazena a resposta de b.
respc=0		#Armazena a resposta de c.
for a in range(1,1000):
	for b in range(a+1,1000):
		c =1000-a-b		#1000 = a+b+c logo c = 1000-a-b.
		if(a*a+b*b==c*c):	#Se a²+b²=c² armazena as respostas.
			respa=a
			respb=b
			respc=c
print(respa*respb*respc)	#Mostra o produto de a, b e c.
